src/AudioToolbox/AudioType.cs

```diff
@@ -663,7 +689,10 @@ public enum AudioChannelLayoutTag : uint { // UInt32 AudioChannelLayoutTag
 		DTS_8_0_B                = (179<<16) | 8,                        
 		DTS_8_1_A                = (180<<16) | 9,                        
 		DTS_8_1_B                = (181<<16) | 9,                        
-		DTS_6_1_D                = (182<<16) | 7,                         
+		DTS_6_1_D                = (182<<16) | 7,
+
+		HOA_ACN_SN3D             = (190U<<16),
+		HOA_ACN_N3D              = (191U<<16),
```

Comment:
It's already inconsistent between enums but maybe new values should follow the newer/correct guidelines, `HoaAcnSN3D` and `HoaAcnN3D` ?